docs/clouds/qemu.md

```diff
@@ -0,0 +1,104 @@
+# QEMU
+
+The following page documents the QEMU cloud integration in pycloudlib.
+
+## Support
+
+Pycloudlib currently supports launching x86-64 Ubuntu images using
+`qemu-system-x86_64` on an x86-64 host.
+
+## Requirements
+
+In addition to requirements specified in `setup.py`, on Ubuntu systems,
+the QEMU cloud requires the following packages:
+
+* qemu-system-x86
+* qemu-utils  # for qemu-img
+* genisoimage  # to create the cloud-init NoCloud seed image
+
+## Configuration and Local Storage
+
+QEMU instances require local directories to store images and instances.
+These directories can be configured in the `pycloudlib.toml` file:
+
+`image_dir` - Directory to look for images when launching instances.
```

Comment:
Can that be a separate refactoring PR when needed? I think each page would need its own parsing as the url, title, filename, kernel-name (if even included...it's not for desktop) are all different.

Reply:
Yes separate PR and iteration on that should be fine and it'd be preparatory work anyway for potential support of other distros anyway so it could be folded into that type of new feature. In the desktop case, we can still curl/wget the image ourselves outside of pycloudlib and feed it into the test framework to exercise the instance lifecycle.